app/config/settings_prod.py

```diff
@@ -49,18 +49,19 @@
 # Application definition
 
 INSTALLED_APPS = [
-    'rest_framework',
-    'rest_framework_gis',
-    'stac_api.apps.StacApiConfig',
-    'config.apps.StacAdminConfig',
     'django.contrib.auth',
     'django.contrib.contenttypes',
     'django.contrib.sessions',
     'django.contrib.messages',
-    'whitenoise.runserver_nostatic',
     'django.contrib.staticfiles',
     'django.contrib.gis',
-    'solo.apps.SoloAppConfig'
+    'rest_framework',
+    'rest_framework_gis',
+    'solo.apps.SoloAppConfig',
+    'storages',
+    'whitenoise.runserver_nostatic',
+    'config.apps.StacAdminConfig',
+    'stac_api.apps.StacApiConfig',
```

Comment:
If this order is required, then I suggest to add a comment about it.

Reply:
Not required, but imho it helps to group by 'standard django > third-party apps > own apps'. I'll add a comment